rules/windows/defense_evasion_masquerading_as_elastic_endpoint_process.toml

```
@@ -0,0 +1,41 @@
+[metadata]
+creation_date = "2020/08/24"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/08/24"
+
+[rule]
+author = ["Elastic"]
+description = """
+A suspicious Elastic endpoint agent parent process was detected. This may indicate a process hollowing or other form of code injection. Verify process details such as hash, command line, network
+connections, file writes, parent and child process details as well.
```

```suggestion
A suspicious Elastic endpoint agent parent process was detected. This may indicate a process hollowing or other form of code injection.
```
I'd move these triage/investigation notes to the `note` field in the rule:

> Verify process details such as hash, command line, network

I think the  rule needs to be linted.
connections, file writes, parent and child process details as well.